### DISCUSSION TITLE
Adjustment of \item Autocompletion

### DISCUSSION THREAD

**Discussion by premiertutors** [James-Yu/LaTeX-Workshop#3809] [Q&A]
Hi Guys,

Is it possible to adjust the autocompletion behaviour that occurs for automatic \item insertion:

> When the current line starts with \item or \item[], hitting Enter automatically adds a newline starting in the same way. For a better handling of the last item, hitting Enter on a line only containing \item or \item[] actually deletes the content of the line. The alt+Enter is bind to the standard newline command. This automatic insertion of \item can be deactivated by setting latex-workshop.bind.enter.key to false.

Specifically, is it possible to replicate this behaviour with a custom item, e.g.

```
\begin{list_enviornment}
  \itemX{first line content}
  \itemX{} %automatically inserted with curser ideally between brackets instead of automatic \item insertion
  \end{list_enviornment}
  ```

**Comment by jlelong** [comment]
Unfortunately, there is no way to robustly customise the automatic insertion of `\item....`. This function is already quite delicate.
